### ISSUE TITLE
Visual distinction between vanila crafting pipe and fuzzy upgraded.

### ISSUE THREAD

**Issue by ZZaphodd** [RS485/LogisticsPipes#969]
LP version : 0.9.3.114
![image](https://cloud.githubusercontent.com/assets/8099421/14576528/dffc5822-03a4-11e6-9e24-823675650828.png)

I see nice blue wavy mark on fuzzy crafting table, but fuzzy upgraded crafting seems exact same like vanila one.

If fuzzy upgraded crafting pipe looks different from vanila one, player might find their mistake easier.

**Comment by SonOfTheStars** [comment]
I vote this :+1:

Could have the Blue Wave Icon too...or some kind of Dots on the side

**Comment by hron84** [comment]
@Bitterholz hint: you can use the topright icon on the first comment to add your reaction.

**Comment by bookerthegeek** [comment]
@hron84 harder to do on a phone.

**Comment by hron84** [comment]
Yeah, GitHub has a poor usability on phone. Sorry, I didn't know you are from mobile.
